src/monitor/group_state_machine.c

```diff
@@ -855,11 +860,10 @@ ProceedGroupStateForPrimaryNode(AutoFailoverNode *primaryNode)
 
 				LogAndNotifyMessage(
 					message, BUFSIZE,
-					"Setting goal state of node %d (%s:%d) to wait_primary "
-					"now that none of the standbys are healthy anymore.",
-					primaryNode->nodeId,
-					primaryNode->nodeHost,
-					primaryNode->nodePort);
+					"Setting goal state of " NODE_FORMAT
+					" to wait_primary now that none of the standbys "
+					"are healthy anymore.",
```

Comment:
nit: leading space for consistency sake

```suggestion
					" to wait_primary now that none of the standbys"
					" are healthy anymore.",
```